### PR TITLE
Fix up a syntax error

### PR DESCRIPTION
**Pull request by Rodrigo-Tenorio**
This should have been called out by a test?

### CODE REVIEW

**Comment by Rodrigo-Tenorio** [comment]
Ok: Tests never checks for timestamps + input detector, so it wasn't covered at all.

**Comment by dbkeitel** [comment]
> Ok: Tests never checks for timestamps + input detector, so it wasn't covered at all.

Can you add such a case?

**Comment by Rodrigo-Tenorio** [comment]
I think I just did. This is one of those cases in which fixtures would make our life easier, but it will work for now (again).

File: pyfstat/make_sfts.py [hunk 1]
```diff
@@ -368,17 +368,17 @@ def _parse_timestamps(self):
...
+
             if self.detectors is not None:
-                ifos_in_ts = list(self.timetamps.keys())
                 ifos_in_detectors = self.detectors.split(",")
-                if np.setdiff1d(ifos_in_ts, ifos_in_detectors).size:
+                if np.setdiff1d(ifos, ifos_in_detectors).size:
                     raise ValueError(
-                        f"Detector names in timestamps dictionary ({ifos_in_ts}) "
+                        f"Detector names in timestamps dictionary ({ifos}) "
                         "are inconsistent with detector names given via keyword ({ifos_in_detectors})."
```
Comment (dbkeitel) [inline pyfstat/make_sfts.py:379]:
```suggestion
                        f"are inconsistent with detector names given via keyword ({ifos_in_detectors})."
```

**Review by dbkeitel — APPROVED** [review]

**Pull request merged**